pootle/apps/pootle_store/models.py

```diff
@@ -356,6 +359,11 @@ class Unit(models.Model, base.TranslationUnit):
                                     db_index=True, related_name='reviewed')
     reviewed_on = models.DateTimeField(db_index=True, null=True)
 
+    # this is calculated from virtualfolders if installed and linked
+    priority = models.FloatField(
+        db_index=True, default=1,
+        validators=[MinValueValidator(0.9)])
```

Comment:
good spot on the typo

validation is always good

i know that mlo uses sub 1 values so ill set to 0.1